filter_plugins/slurm_conf.py

```diff
@@ -58,9 +74,16 @@ def _group_numbers(numbers):
         prev = v
     return ','.join(['{}-{}'.format(u[0], u[-1]) if len(u) > 1 else str(u[0]) for u in units])
 
+def error(condition, msg):
+    """ Raise an error if condition is not True """
+    
+    if not condition:
+        raise errors.AnsibleFilterError(msg)
+
 class FilterModule(object):
 
     def filters(self):
         return {
-            'group_hosts': group_hosts
```

Comment:
That looks like its backwards incompatible? Could we keep old and new easily enough?

Reply:
The problem is the filter name is massively confusing when used in the templating which also used `group_hosts` as a variable containing the list of hosts in that group. I can't see why backward compat is required as I can't see a use-case for another playbook using this filter.